src/rpc/claimtrie.cpp

```diff
@@ -382,16 +382,16 @@ UniValue getclaimbyid(const UniValue& params, bool fHelp)
                         "  \"n\"                   (numeric) vout value\n"
                         "  \"amount\"              (numeric) txout value\n"
                         "  \"effective amount\"    (numeric) txout amount plus amount from all supports associated with the claim\n"
-                        "  \"height\"              (numeric) the height of the block in which this claim transaction is located\n"
```

Comment:
Just curious why you're re-ordering here.  Is this related to something about the doc generation?

Reply:
"valid at height" was between the supports array declaration and its definition. That was definitely wrong. I moved "height" as well just so that it matched the code filling it in.

Reply:
:+1: Thanks